source/_integrations/calendar.markdown

```diff
@@ -180,3 +180,16 @@ data:
   start_date: "2022-10-01"
   end_date: "2022-10-02"
 ```
+
+Recent changes to Calendars in Home Assistant mean that zero duration Calendar events
+are no longer allowed. The following would create a one minute long event starting "now".
+
+```yaml
```

Comment:
This code-fenced block contains Jinja language, which conflicts with the Liquid language used by Jekyll that generates this website. To prevent issues, please wrap the code-block in `{% raw %}` and `{% endraw %}`.

Reply:
Done.